runtime/help/plugins.md

```diff
@@ -355,85 +439,92 @@ end
+* [net](https://pkg.go.dev/net)
+* [math](https://pkg.go.dev/math)
+* [math/rand](https://pkg.go.dev/math/rand)
+* [os](https://pkg.go.dev/os)
+* [runtime](https://pkg.go.dev/runtime)
+* [path](https://pkg.go.dev/path)
+* [filepath](https://pkg.go.dev/filepath)
+* [strings](https://pkg.go.dev/strings)
+* [regexp](https://pkg.go.dev/regexp)
+* [errors](https://pkg.go.dev/errors)
+* [time](https://pkg.go.dev/time)
+* [unicode/utf8](https://pkg.go.dev/unicode/utf8)
+* [archive/zip](https://pkg.go.dev/archive/zip)
+* [net/http](https://pkg.go.dev/net/http)
+
+The following functions from the [go-humanize](Humanize) package are also
+available:
+
+* `humanize`:
+    - `Bytes(s uint64) string`:
+       produces a human readable representation of an SI size.
+    - `Ordinal(x int) string`:
+        gives you the input number in a rank/ordinal format.
+
+[The Lua standard library][Lua] is also available to plugins, though it is
+rather small.
+
+[Humanize]: https://pkg.go.dev/github.com/dustin/go-humanize
+[Lua]: https://www.lua.org/manual/5.1/manual.html#5
 
 ## Adding help files, syntax files, or colorschemes in your plugin
 
 You can use the `AddRuntimeFile(name string, type config.RTFiletype,
                                 path string)`
 function to add various kinds of files to your plugin. For example, if you'd
 like to add a help topic to your plugin called `test`, you would create a
-`test.md` file, and call the function:
+`test.md` file and call the function:
 
 ```lua
 config = import("micro/config")
 config.AddRuntimeFile("test", config.RTHelp, "test.md")
 ```
 
 Use `AddRuntimeFilesFromDirectory(name, type, dir, pattern)` to add a number of
-files to the runtime. To read the content of a runtime file use
+files to the runtime. To read the content of a runtime file, use
 `ReadRuntimeFile(fileType, name string)` or `ListRuntimeFiles(fileType string)`
 for all runtime files. In addition, there is `AddRuntimeFileFromMemory` which
 adds a runtime file based on a string that may have been constructed at
 runtime.
 
 ## Default plugins
 
-There are 6 default plugins that come pre-installed with micro. These are
-
-* `autoclose`: automatically closes brackets, quotes, etc...
-* `comment`: provides automatic commenting for a number of languages
-* `ftoptions`: alters some default options (notably indentation) depending on the filetype
-* `linter`: provides extensible linting for many languages
-* `literate`: provides advanced syntax highlighting for the Literate
-   programming tool.
-* `status`: provides some extensions to the status line (integration with
-   Git and more).
-* `diff`: integrates the `diffgutter` option with Git. If you are in a Git
-   directory, the diff gutter will show changes with respect to the most
-   recent Git commit rather than the diff since opening the file.
+There are 7 default plugins that come pre-installed with micro. These are
+
+* `autoclose`:
+   automatically closes brackets, quotes, etc...
```

Comment:
I'm not sure that moving these to separate lines here really makes it more readable.

Reply:
I think I moved them because for consistency with the function descriptions. Now that I look at it, it really does not improve anything.